File: Frontend-Shopping/src/app/shared/models/filter-options.ts
export class FilterOptions {
    name: string = null;
    brandId: number[] = [];
    departmentId: number[] = [];
    categoryId: number[] = [];
    minPrice: number = 0;
    maxPrice: number = 999999999999999999999;

    constructor(name: string) {
        this.name = name;
    }
}

File: Backend-Shopping/DataService/dtos/Filter/FilterOptions.cs
namespace DataService.dtos.Filter
{
    public class FilterOptions
    {
        public string Name { get; set; }
        public int[] BrandId { get; set; }
        public int[] DepartmentId { get; set; }
        public int[] CategoryId { get; set; }
        public double MinPrice { get; set; } = 0;
        public double MaxPrice { get; set; } = 99999999999999999;
    }
}